CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+
+### [14.1.0](https://github.com/mojaloop/ml-api-adapter/compare/v14.0.4...v14.0.5) (2023-11-08)
+
+### Features
+
+* **mojaloop/#3574:** add FX Transfers requests ([#3574](https://github.com/mojaloop/ml-api-adapter/issues/3574))
+
+
```

Comment:
May be we can deal with this changelog later when we want to merge to main branch.